app/model/AuditContext.scala

```diff
@@ -50,17 +49,17 @@ object RoutingReason {
   val HAS_SA_UTR = Reason("has-sa-utr")
   val HAS_REGISTERED_FOR_2SV = Reason("has-registered-for-2sv")
   val HAS_STRONG_CREDENTIALS = Reason("has-strong-credentials")
-  val HAS_ONLY_ONE_ENROLMENT = Reason("has-only-one-enrolment")
   val HAS_INDIVIDUAL_AFFINITY_GROUP = Reason("has-individual-affinity-group")
   val HAS_ANY_INACTIVE_ENROLMENT = Reason("has-any-inactive-enrolment")
   val AFFINITY_GROUP_AVAILABLE = Reason("affinity-group-available")
+  def HAS_ENROLMENTS(enrolmentCategories : Set[EnrolmentCategory]) = Reason(s"""has:${enrolmentCategories.map(_.enrolmentCategoryName).mkString(",")}""")
```

Comment:
please check if this format is allowed in Graphite (it creates a directory on fs with this name)